### PR TITLE
Fix s11 file read

### PR DESCRIPTION
**Pull request by nmahesh1412**
small fixes to read the S11 file. It will not read any of the formats txt ot csv

### CODE REVIEW

File: src/edges_analysis/analysis/levels.py [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ def get_s11_paths(
                 fls.append(p)
 
             return fls
-        if s11_path.endswith("csv"):
+        if (os.path.isfile(s11_path)) and (not s11_path.endswith("s1p")):
```
Comment (nmahesh1412) [inline src/edges_analysis/analysis/levels.py:1610]: @steven-murray when I was writing this I realized that s11_path.is_dir() won't work.

Comment (nmahesh1412) [reply src/edges_analysis/analysis/levels.py:1610]: so I am confused how didnt we get an error in line 1618 (s11_path.is_dir()) ?

Comment (steven-murray) [reply src/edges_analysis/analysis/levels.py:1610]: You'll need to make sure you convert the `s11_path` to a `pathlib.Path` instance (i.e. do `s11_path = Path(s11_path)`) before you can use the `is_dir` method on it

Comment (steven-murray) [reply src/edges_analysis/analysis/levels.py:1610]: But I can accept this as is.

**Review by steven-murray — APPROVED** [review]
Thanks @nmahesh1412 !

**Pull request merged**